tests/integration/targets/aci_contract_subject/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 # Test code for the ACI modules
```

Comment:
did you execute the test file? since I am getting errors which would indicate test file has not been successfully run